### ISSUE TITLE
There is a dependency error with Fastify 4.16.0

### ISSUE THREAD

**Issue by patoi** [fastify/fastify#4703]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the bug has not already been reported

### Fastify version

4.16.0

### Node.js version

18.16.0

### Operating system

Linux

### Operating system version (i.e. 20.04, 11.3, 10)

Ubuntu 20.04

### Description

Updating from 4.15.0 to 4.16.0, and I got a dependency error when I starting my app.
There is a reference to `fast-json-stringify/lib/standalone`  in `fastify/lib/error-serializer.js` but this is in the devDependencies of Fastify.

```
➜ node --max-http-header-size=81920 server
node:internal/modules/cjs/loader:1075
  const err = new Error(message);
              ^

Error: Cannot find module 'fast-json-stringify/lib/standalone'
Require stack:
- /home/app/node_modules/.pnpm/fastify@4.16.0/node_modules/fastify/lib/error-serializer.js
- /home/app/node_modules/.pnpm/fastify@4.16.0/node_modules/fastify/lib/error-handler.js
- /home/app/node_modules/.pnpm/fastify@4.16.0/node_modules/fastify/lib/reply.js
- /home/app/node_modules/.pnpm/fastify@4.16.0/node_modules/fastify/fastify.js
- /home/app/server/server.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)
    at Hook._require.Module.require /home/app/node_modules/.pnpm/require-in-the-middle@6.0.0/node_modules/require-in-the-middle/index.js:81:25)
    at require (node:internal/modules/cjs/helpers:110:18)
    at Object.<anonymous> (/home/app/node_modules/.pnpm/fastify@4.16.0/node_modules/fastify/lib/error-serializer.js:5:28)
    at Module._compile (node:internal/modules/cjs/loader:1254:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Module._load (node:internal/modules/cjs/loader:958:12)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at Hook._require.Module.require (/home/app/node_modules/.pnpm/require-in-the-middle@6.0.0/node_modules/require-in-the-middle/index.js:101:39) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/app/node_modules/.pnpm/fastify@4.16.0/node_modules/fastify/lib/error-serializer.js',
    '/home/app/node_modules/.pnpm/fastify@4.16.0/node_modules/fastify/lib/error-handler.js',
    '/home/app/node_modules/.pnpm/fastify@4.16.0/node_modules/fastify/lib/reply.js',
    '/home/app/node_modules/.pnpm/fastify@4.16.0/node_modules/fastify/fastify.js',
    /home/app/server/server.js'
  ]
}

Node.js v18.16.0
```

### Steps to Reproduce

Starting a Fastify app.

**Comment by patoi** [comment]
I think the https://github.com/fastify/fastify/issues/4702 is related to this.

**Comment by mcollina** [comment]
Fixed in main, releasing v4.16.1 asap.

(Updating your dependency tree would have fixed it as well).

**Comment by patoi** [comment]
I think it is not fixed with 4.16.2, because https://github.com/fastify/fastify/commit/c922d63a4e31cf6e91bf74fa43c3662b496827d0 has not moved module to `dependencies` from `devDependencies`

**Comment by mcollina** [comment]
Fixed in v4.16.3.
